common/_templates/page.html

```diff
@@ -9,16 +9,15 @@
 
   <!-- Helpfulness -->
   <div id="helpful-position"></div>
-  <div id="helpful" class="helpfulness-container fixed" style="display:none;">
+  <div id="helpful" class="helpfulness-container" style="display:none;">
     <div class="helpfulness">
       <div class="helpfulness-form">
-        <h2>Was this page helpful?</h2>
-        <span>Your feedback about this content is important. Let us know what you think.</span>
+        <h2>Is this page helpful?</h2>
         <a id="helpfulness-btn-yes" href="javascript:;" class="common-btn first">Yes</a>
         <a id="helpfulness-btn-no" href="javascript:;" class="common-btn">No</a>
       </div>
       <div class="helpfulness-form-no" style="display:none;">
-        <h2>Was this page helpful?</h2>
+        <h2>Is this page helpful?</h2>
         Sorry this wasn't helpful.
```

Comment:
Possibly. We are using the same text as azure.com.

From: Eilon Lipton [mailto:notifications@github.com]
Sent: Monday, May 23, 2016 11:04 PM
To: aspnet/Docs <Docs@noreply.github.com>
Cc: Samir Patel <samir.patel@neudesic.com>; Mention <mention@noreply.github.com>
Subject: Re: [aspnet/Docs] Helpfulness (#1279)


In common/_templates/page.html<https://github.com/aspnet/Docs/pull/1279#discussion_r64332757>:

>          <a id="helpfulness-btn-yes" href="javascript:;" class="common-btn first">Yes</a>

>          <a id="helpfulness-btn-no" href="javascript:;" class="common-btn">No</a>

>        </div>

>        <div class="helpfulness-form-no" style="display:none;">

> -        <h2>Was this page helpful?</h2>

> +        <h2>Is this page helpful?</h2>

>          Sorry this wasn't helpful.

Should the answer(s) also be in present tense?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Docs/pull/1279/files/b6e86cf0b1a450c62d29f4c4e947d8d3c74ce1dd#r64332757>


Confidentiality Notice: This email and any attachments are confidential. If you have received this in error, please let us know by email reply and delete the email and all attachments from your system.